Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN cargo build -p sqld --release
 
 # runtime
 FROM debian:bullseye-slim
+RUN apt update
```

Comment:
How much bigger does this make the image?

Reply:
18MB
```
REPOSITORY                                  TAG                                     IMAGE ID       CREATED          SIZE
ghcr.io/tursodatabase/libsql-server         main                                    73ff550254d1   16 minutes ago   115MB
ghcr.io/tursodatabase/libsql-server-devel   pr-513-124f428                          55502143c1c6   14 hours ago     133MB
```